dbsnp-importer/src/test/resources/dbsnp-data.sql

```diff
@@ -162,3 +162,65 @@ INSERT INTO dbsnp_shared.obsvariation VALUES (20,'G/A','2003-02-22 01:07:00','20
 INSERT INTO dbsnp_shared.obsvariation VALUES (34,'-/G','2003-02-22 01:07:00','2003-07-17 14:33:00',15,NULL,'-/G');
 INSERT INTO dbsnp_shared.obsvariation VALUES (11680,'TCGG/-','2003-04-16 09:16:00','2003-07-17 14:33:00',238,NULL,'TCGG/-');
 INSERT INTO dbsnp_shared.obsvariation VALUES (13946,'TCAGG/-','2003-04-16 15:12:00','2003-07-17 14:33:00',570,NULL,'TCAGG/-');
+
+-- ss_id, rs_id, batch_id, batch_name, hgvs_c_string, hgvs_c_start, hgvs_c_stop, reference_c, hgvs_t_string, hgvs_t_start, hgvs_t_stop, reference_t, alternate, alleles, contig_name, contig_start, contig_end, loc_type, chromosome, chromosome_start, chromosome_end, hgvs_c_orientation, hgvs_t_orientation, snp_orientation, contig_orientation, subsnp_orientation, genotypes_string, freq_info, load_order
+INSERT INTO dbsnp_variant_load_d8c757988871529f37061fa9c79477a5 VALUES(24828977, 13713751, 11825, 'CHICKEN_SNPS_BROILER', 'NC_006088.4:g.14157380_14157381insA', 14157380, 14157381, '-', 'NW_001471565.3:g.2723319_2723320insA', 2723319, 2723320, '-', 'T', '-/ACAG', 'NW_001471565.3', 2723319, 2723320, 3, '1', 14157380, 14157381, -1, -1, -1, 1, 1, '-/-,ACAG/ACAG', '[{"pop_id" : 1324, "pop_name" : "RBLS", "freq_info" : [{"allele" : "ACAG", "cnt" : 2.0, "freq" : 0.5}, {"allele" : "-", "cnt" : 2.0, "freq" : 0.5}]}]', 19403);
```

Comment:
If we are going to replace the old readers by the one using the materialized view, then we should delete the obsolete test data (unless we still need it for other reader)

Reply:
we still need it for the sample reader tests